kong/api/routes/metadata.lua

```diff
@@ -0,0 +1,25 @@
+local cjson = require "cjson"
+local utils = require "kong.tools.utils"
+
+local kong = kong
+local ngx = ngx
+
+return {
+  ["/metadata/plugins/:name"] = {
+    GET = function(self, db, helpers, parent)
+      ngx.log(ngx.DEBUG, "self.params: ", cjson.encode(self.params))
+      local name = self.params.name
+      if not name then
+        return kong.response.exit(400, { message = "Bad query" })
+      end
+
+      -- TODO: either unload module or use loadfile() instead
```

Comment:
`load_module_if_exists` wraps `require` and is the way used for loading plugin modules elsewhere, so it's probably best to keep using it and unload the module from `package.loaded`.